docs/es/FAQ.es.md

```diff
@@ -0,0 +1,3 @@
+# Preguntas frecuentes
+
+Imaginemos que usted es un nuevo desarrollador encargado de agregar una nueva función o corregir un error. Empiezas a leer el código pero no sabes por dónde empezar. Necesitas encontrar el lugar correcto para comenzar. ¡Te tenemos cubierto!
```

Comment:
Maybe exists a better translate or phrase to say "Te tenemos cubierto" in spanish